src/main.js

```diff
@@ -82,14 +85,14 @@ function toggleHomeButton() {
 
 // Iteration 3:
 var makeNewBookButton = document.querySelector('.create-new-book-button');
-makeNewButton.type = "button";
```

Comment:
I want to revisit whether or not we need to reassign this button type. I think it will work without this line.